terraform/examples/aws-complete/aws

```diff
@@ -0,0 +1 @@
+../../aws
```

Comment:
In the base PR I actually switched to using GitHub paths with a branch, I actually forgot to point it back to main before landing:

`github.com/xmtp/xmtpd-infrastructure//terraform/aws/xmtp-validation-service?ref=rich-tf-validation-svc`

I wonder if that would be easier to work off of

Reply:
That's the right place to leave the repo. It gets a little weird when you're trying to run `terraform validate` and you have any changes in the dependent modules.

You could push and then point at your current branch, but then you have to point back to main at some point...at which point the linter will fail. Almost need to break the PR into two pieces, one to update the module and one to update the dependency.

Reply:
I'll point it to `main` in the next PR, when `main` has stuff in the right place